src/node_crypto.cc

```diff
@@ -4525,13 +4527,14 @@ void SignBase::CheckThrow(SignBase::Error error) {
 static bool ApplyRSAOptions(const ManagedEVPPKey& pkey,
                             EVP_PKEY_CTX* pkctx,
                             int padding,
-                            int salt_len) {
+                            const Maybe<int>& salt_len) {
   if (EVP_PKEY_id(pkey.get()) == EVP_PKEY_RSA ||
-      EVP_PKEY_id(pkey.get()) == EVP_PKEY_RSA2) {
+      EVP_PKEY_id(pkey.get()) == EVP_PKEY_RSA2 ||
```

Comment:
what's 'rsa2'? it doesn't need a specific js-side key type like rsa-pss does?

Reply:
`EVP_PKEY_RSA2` is the OpenSSL-ish way to express an old and barely used OID for RSA keys (2.5.8.1.1). `EVP_PKEY_base_id` returns `EVP_PKEY_RSA` for both `EVP_PKEY_RSA` and `EVP_PKEY_RSA2`, but *not* for `EVP_PKEY_RSA_PSS`.